client/actions/main.ts

```diff
@@ -237,6 +237,7 @@ const save = (original, updates, withConfirmation = true) => (
         switch (itemType) {
         case ITEM_TYPE.EVENT:
             promise = dispatch(eventsUi.save(original, updates, confirmation));
+            confirmation = false;
```

Comment:
Looking further at the code of this function, this solution won't work either.
If the user clicks `cancel` in the Ignore/Cancel/Save modal.
See the comment here: https://github.com/superdesk/superdesk-planning/blob/1d07aa9d09927e29c726741808a56fd3c9ccef2c/client/actions/main.ts#L258